src/engraving/rendering/dev/arpeggiolayout.cpp

```diff
@@ -151,6 +151,8 @@ void ArpeggioLayout::layoutOnEdit(Arpeggio* item, LayoutContext& ctx)
 //---------------------------------------------------------
 void ArpeggioLayout::symbolLine(const IEngravingFontPtr& f, Arpeggio::LayoutData* data, SymId end, SymId fill)
 {
+    data->symbols.clear();
```

Comment:
Looks like this will resolve #19083, right?

Reply:
yes (